lib/logstash/inputs/http.rb

```diff
@@ -75,11 +83,20 @@ def register
   end # def register
 
   def run(queue)
```

Comment:
Another point here, _Basic Auth_ is the MVP we want but in a future revision we could also support oAuth style tokens,  and rack libraries exist for that https://github.com/nov/rack-oauth2